pusher-platform-core/src/integrationTest/kotlin/com/pusher/platform/test/describeConnected.kt

```diff
@@ -0,0 +1,35 @@
+package com.pusher.platform.test;
+
+import org.jetbrains.spek.api.dsl.Spec
+import org.jetbrains.spek.api.dsl.SpecBody
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.xdescribe
+import java.net.HttpURLConnection
+import java.net.URL
+import javax.net.ssl.HttpsURLConnection
+
+/**
+ * Skips test if the url is not reachable.
+ */
+fun Spec.describeWhenReachable(uri: String, description: String, body: SpecBody.() -> Unit) = if (isReachable(uri)) {
+    describe(description, body)
+} else {
+    xdescribe(description, "Can't reach $uri", body)
+}
+
+private fun isReachable(uri: String) : Boolean {
```

Comment:
- First two lines disable SSL
 - The second part is verifying if the connection is available (hence the connect+disconnect), if not it disables the test as it can be seen here:
https://travis-ci.org/pusher/pusher-platform-android/jobs/367190509#L2176